main.go

```diff
@@ -83,5 +83,6 @@ func LoadData() [][]string {
 	cr := csv.NewReader(r)
 	records, _ := cr.ReadAll()
 	log.Print(len(records))
+	r.Close()
```

Comment:
Just chiming in because I saw this:  you probably want `defer r.Close()` to let the runtime handle closing the reader.

Reply:
thx. :-D